console/billing/quickstart.mdx

```diff
@@ -0,0 +1,76 @@
+---
+meta:
+  title: Billing - Quickstart
+  description: This page shows you how to get started with the billing of your Scaleway account.
```

Comment:
```suggestion
  description: This page shows you how to get started with the billing for your Scaleway account.
```

Reply:
you should commit these kind of small edits right away if you are sure you want to keep them :) Otherwise it can get confusing for reviewers